binr/blob/main.c

```diff
@@ -1,10 +1,12 @@
 /* radare - LGPL - Copyright 2012-2019 - pancake */
 
 #include <r_main.h>
+#include <r_util.h>
 
 int main(int argc, char **argv) {
 	int rc = 1;
-	RMain *m = r_main_new (argv[0]);
+	const char *prog_name = r_file_basename (argv[0]);
```

Comment:
BSD systems don't need argv0 to get the program name. And this is probably a better way to go for this. afaik same is for windows and haiku

* https://man.openbsd.org/getprogname.3

Reply:
What do you mean? I think argv[0] is quite standard across systems... it seems it is in the C standard that argv[0] contains the program name.